app/src/main/java/com/saucelabs/mydemoapp/android/view/activities/MainActivity.java

```diff
@@ -322,8 +319,7 @@ private void handleMenuClick(int position) {
 				startActivity(new Intent(this, VirtualUsbActivity.class));
 				break;
 			case 9:
-				TestFairy.setFeedbackOptions(new FeedbackOptions.Builder().build());
-				TestFairy.showFeedbackForm();
+				// FIXME: Remove Case ?
```

Comment:
Yes, we should remove this and the next case and the menu items for removed activities from the sidebar entirely.